src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -5958,4 +5958,26 @@ inline void compute_sum_24_host(__m256d *p, __m256d *pSumR, __m256d *pSumG, __m2
     pSumB[0] = _mm256_add_pd(_mm256_add_pd(p[4], p[5]), pSumB[0]); //add 8B values and bring it down to 4
 }
 
+inline void compute_transpose4x8_avx(__m256 *pSrc, __m128 *pDst)
```

Comment:
Calling the function _avx but mixing src as avx, dst as sse is not the best way. Why are we mixing vector lengths here?